github-dark.css

```diff
@@ -2015,8 +2015,7 @@
   .js-current .standalone a, .menu-item:hover,
   .Box-row--focus-gray.navigation-focus, .team-listing .is-open.root-team,
   .Box-row--hover-gray:hover, table.files tr[aria-selected="true"] td,
-  .label-select-menu .select-menu-item[aria-selected="true"],
-  .label-select-menu .select-menu-item.navigation-focus {
```

Comment:
This removal seems unintended

Reply:
Good one, all that move from here to there to the other :)

Should be alright now.